packages/doc/content/components/components/progress/progress-web.mdx

```diff
@@ -0,0 +1,27 @@
+### Usage
+
+```javascript reactLive=true
```

Comment:
The right prop is `type` but `reactLive` is the default prop, so it's no necessary here

Reply:
updated! (and the CLI template)